f3read.c

```diff
@@ -164,6 +164,8 @@ static void validate_file(const char *path, int number,
 
 	/* Obtain initial time. */
 	assert(!gettimeofday(&t1, NULL));
+	pt1 = t1;
+	pt1.tv_sec -= 1000;
```

Comment:
I'm sorry, I glossed over the fact that you tested on an x86-32 machine. You are right.